Source/WTF/wtf/HexNumber.cpp

```diff
@@ -31,6 +31,13 @@ namespace WTF {
 
 namespace Internal {
 
+static const std::array<Latin1Character, 16>& hexDigitsForMode(HexConversionMode mode)
+{
+    static constexpr std::array<Latin1Character, 16> lowercaseHexDigits { '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f' };
```

Comment:
I see you using `std::to_array<>` elsewhere. Why not here?

Reply:
Seems nice to be explicit about the 16 hex digits here